projects/ngrx.io/content/guide/effects/testing.md

```diff
@@ -14,14 +14,14 @@ import { TestBed } from '@angular/core/testing';
 import { provideMockActions } from '@ngrx/effects/testing';
 import { ReplaySubject } from 'rxjs/ReplaySubject';
 import { hot, cold } from 'jasmine-marbles';
-import { Observable } from 'rxjs';
+import { Subject } from 'rxjs';
```

Comment:
This should be broken up into two separate examples, as `Observable<any>` is sufficient for using marble diagrams.